### DISCUSSION TITLE
Does NativePHP support building for android, or ios?

### DISCUSSION THREAD

**Discussion by quiet-observer** [NativePHP/laravel#303] [Q&A] [answered]
From what I read it seems you can only build for mac, win, linux, or am I mistaken?

**Comment by mpociot** [comment, marked as answer]
That's right, mobile app development is potentially on the roadmap but as of right now, we focus on desktop apps.

**Reply by quiet-observer** [reply]
Awesome looking forward to it. Keep up the good work. I have a very useful Laravel codebase and would love for it to be cross-platform across all desktops, and mobile devices. Would be great, otherwise I have to use these other technologies bouncing back and forth between this framework, and the next. Laravel is very solid.

**Reply by simpleittools** [reply]
Congrats on the Windows functionality.
Android and iOS would be amazing.

**Comment by simonhamp** [comment]
[Yes, it's here!](https://nativephp.com/mobile)
